src/components/viz-pane/renderer.tsx

```diff
@@ -69,8 +69,11 @@ export default class VizPane extends React.Component<Props> {
         onChange={this.handleChange}
         paneStyle={{ display: 'flex' }}
         onDragFinished={() => {
-          if (this.props.debugPaneSize === 25) {
+          if (this.props.debugPaneSize === LAYOUT.MinPaneSize) {
             this.props.setDebugPaneSize(60);
```

Comment:
Why not just write a formula instead of setting a magic number?

Reply:
Should I write something like generateNewPos = (prev , offset) => prev+offset

Reply:
No, just `LAYOUT.MinPaneSize + 35`

Reply:
Okay